inventory/sample/group_vars/etcd.yml

```diff
@@ -18,6 +18,13 @@
 # This affects ETCD_PEER_CLIENT_CERT_AUTH variable
 # etcd_peer_client_auth: true
 
+## Container runtime
+## docker for docker, crio for cri-o and containerd for containerd.
+## If this is not set, container manager will be inherited from the Kubespray defaults 
```

Comment:
Please remove unnecessary space at the end of this line to solve ansible-lint issue:
https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/2099430645#L222
```
WARNING  Listing 1 violation(s) that are fatal
inventory/sample/group_vars/etcd.yml:23: yaml trailing spaces (trailing-spaces)
```

Reply:
done